elftools/elf/relocation.py

```diff
@@ -182,6 +193,16 @@ def get_relocation(self, n):
             self._cached_relocations = list(self.iter_relocations())
         return self._cached_relocations[n]
 
+
+class RelrRelocationSection(Section, RelrRelocationTable):
```

Comment:
I'd rather use composition than multiple inheritance, unless there's a strong reason to do the latter.

Reply:
It follows the same style already used by `RelocationSection`.